application/common/tizen/signature_parser.cc

```diff
@@ -111,6 +111,34 @@ xmlNsPtr GetNamespace(const xmlNodePtr node, const char* expected_href) {
 
 namespace xwalk {
 namespace application {
+int hexToInt(char a) {
+  if (a >= '0' && a <= '9') return a - '0';
+  if (a >= 'A' && a <= 'F') return a - 'A' + 10;
+  if (a >= 'a' && a <= 'f') return a - 'a' + 10;
+  return -1;
```

Comment:
I'd suggest to copy the implemation of [src/third_party/WebKit/Source/core/inspector/JSONParser.cpp] (https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/core/inspector/JSONParser.cpp&q=hextoInt&sq=package:chromium&l=258&ct=rc&cd=6&dr=CSs). 

Note, please change to chromium coding style, not the WebKit/Blink style.